src/components/application_manager/include/application_manager/message.h

```diff
@@ -63,7 +63,8 @@ enum ProtocolVersion {
   kV1 = 1,
   kV2 = 2,
   kV3 = 3,
-  kV4 = 4
+  kV4 = 4,
+  kV5 = 5
```

Comment:
Issue was already entered [here](https://github.com/smartdevicelink/sdl_core/issues/1698), though I will see if I can include in this PR easily